pkgs/development/python-modules/pijuice/patch-shebang.diff

```diff
@@ -0,0 +1,9 @@
+diff --git a/Software/Source/src/pijuice_cli.py b/Software/Source/src/pijuice_cli.py
+index c366fee..37af383 100644
+--- a/src/pijuice_cli.py
++++ b/src/pijuice_cli.py
+@@ -1,3 +1,4 @@
++#!/usr/bin/python3
```

Comment:
Wrong shebang? /usr/bin/python3 doesn't exist on nix

Reply:
Agreed, but there's no shebang at all in this script which means the `wrapPythonProgram` hook doesn't run.  By adding this shebang (even though it's wrong) it allows it to get picked up by the hook.

Reply:
Nice hack!
Don't forget to explain it as a comment on `default.nix`.